sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suite.scala

```diff
@@ -121,7 +152,16 @@ class HiveThriftServer2Suite extends FunSuite with Logging {
   }
 
   def startThriftServer(
+      thriftPort: Int,
+      serverStartTimeout: FiniteDuration = 1.minute,
+      httpMode: Boolean = false)(
+      f: => Unit): Unit = {
+    startThriftServerWithUIPort(thriftPort, randomListeningPort, serverStartTimeout, httpMode)(f)
+  }
+
+  def startThriftServerWithUIPort(
```

Comment:
As we discussed, reuse  startThriftServer will cause `multiple overloaded alternatives of method startThriftServer define default arguments`. I will modify the `startThriftServer` function directly.